HLTriggerOffline/Exotica/src/HLTExoticaValidator.cc

```diff
@@ -42,6 +42,18 @@ HLTExoticaValidator::HLTExoticaValidator(const edm::ParameterSet& pset) :
 
 HLTExoticaValidator::~HLTExoticaValidator()
 {
+    // Dump trigger results
+    std::cout << std::endl;
```

Comment:
This use of cout is not allowed in production code. Please switch to message logger if this is to ever be used in centrally run workflows.